packages/core/content-releases/admin/src/pages/DetailsPage/Release.tsx

```diff
@@ -0,0 +1,188 @@
+import * as React from 'react';
+
+import {
+  Button,
+  ContentLayout,
+  EmptyStateLayout,
+  Flex,
+  HeaderLayout,
+  IconButton,
+  Link,
+  Popover,
+  Typography,
+} from '@strapi/design-system';
+import { ArrowLeft, EmptyDocuments, More, Pencil, Trash } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+
+const PopoverButton = styled(Flex)`
+  align-self: stretch;
+`;
+
+const PencilIcon = styled(Pencil)`
+  width: 16px;
+  height: 16px;
+  path {
+    fill: ${({ theme }) => theme.colors.neutral600};
+`;
+
+const TrashIcon = styled(Trash)`
+  width: 16px;
+  height: 16px;
+  path {
+    fill: ${({ theme }) => theme.colors.danger600};
+`;
+
+const ReleaseInfoWrapper = styled(Flex)`
+  align-self: stretch;
+  border-radius: 0 0 4px 4px;
+  border-top: 1px solid ${({ theme }) => theme.colors.neutral150};
+`;
+
+const ReleasePage = () => {
+  const [isPopoverVisible, setIsPopoverVisible] = React.useState(false);
+  const moreButtonRef = React.useRef<HTMLButtonElement>(null!);
+  const { formatMessage } = useIntl();
+  // TODO: get the title from the API
+  const title = 'Release title';
+
+  const totalEntries = 0; // TODO: replace it with the total number of entries
+  const days = 0; // TODO: replace it with the number of days since the release was created
+  const createdBy = 'John Doe'; // TODO: replace it with the name of the user who created the release
```

Comment:
This is actually really interesting, because if we are getting this information from the API we also need some kind of "loading state" here, do you want to wait to implement that for when we have the `findOne` endpoint?

Reply:
Yes, exactly